src/core/AutoRest.Core/Properties/Resources.resx

```diff
@@ -195,6 +195,41 @@
   <data name="MsdnReferencesDiscouraged" xml:space="preserve">
     <value>For better generated code quality, remove all references to "msdn.microsoft.com".</value>
   </data>
+  <data name="APIVersionFormatIsNotValid" xml:space="preserve">
+    <value>API Version must be in the format: yyyy-MM-dd, optionally followed by -preview, -alpha, -beta, -rc, -privatepreview.</value>
+  </data>
+  <data name="HttpVerbIsNotValid" xml:space="preserve">
+    <value>Permissible values for HTTP Verb are delete,get,put,patch,head,options,post. </value>
+  </data>
+  <data name="ResourceModelIsNotValid" xml:space="preserve">
+    <value>The id, name, type, location and tags properties of the Resource must be present with id, name and type as read-only</value>
+  </data>
+  <data name="ResourceIsMsResourceNotValid" xml:space="preserve">
+    <value>A 'Resource' definition must have x-ms-azure-resource extension enabled and set to true.</value>
+  </data>
+  <data name="XmsClientNameInValid" xml:space="preserve">
+    <value>Value of 'x-ms-client-name' cannot be the same as the property/model name.</value>
+  </data>
+  <data name="DeleteMustHaveEmptyBody" xml:space="preserve">
+    <value>'Delete' operation cannot have parameters in body.</value>
+  </data>
+  <data name="SkuModelIsNotValid" xml:space="preserve">
+    <value>Sku Model is not valid. A Sku model must have name property. It can also have tier, size, family, capacity as optional properties.</value>
+  </data>
+  <data name="PutGetPatchResponseInvalid" xml:space="preserve">
+    <value>{0} has different responses for PUT/GET/PATCH operations. The PUT/GET/PATCH operations must have same schema response.</value>
+  </data>
+  <data name="OperationsAPINotImplemented" xml:space="preserve">
+    <value>Operations API must be implemented for the service.</value>
+  </data>
+  <data name="TrackedResourceIsNotValid" xml:space="preserve">
```

Comment:
This rule does not accurately represent whether the tracked resource is valid or not. So the rule name /description can be misleading. Ex- The listByResourceGroup and listBySubscription need not be present for tracked nested resources. So we should Run 2. and 3. checks only for tracked parents.